libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSourceTest.kt

```diff
@@ -949,6 +955,153 @@ class MapboxNavigationViewportDataSourceTest {
         verify { getScreenBoxForFraming(mapSize, followingPadding) }
     }
 
+    @Test
+    fun `verify frame - location + route + progress + reset route`() {
+        every {
+            getPitchFallbackFromRouteProgress(
+                viewportDataSource.options.followingFrameOptions.pitchNearManeuvers.enabled,
+                viewportDataSource.options.followingFrameOptions.pitchNearManeuvers
+                    .triggerDistanceFromManeuver,
+                viewportDataSource.options.followingFrameOptions.defaultPitch,
+                any()
+            )
+        } returns ZERO_PITCH
+
+        val stepProgress = mockk<RouteStepProgress> {
+            every { distanceRemaining } returns 123f
+        }
+        val legProgress = mockk<RouteLegProgress> {
+            every { currentStepProgress } returns stepProgress
+        }
+        every { routeProgress.currentLegProgress } returns legProgress
+        every { routeProgress.route } returns route
+
+        val location = createLocation()
+
+        // following mocks
```

Comment:
Could we name the variable differently (e.g. `mockedFollowingCameraOptions`) and remove the comment?

The same thing applies to other comments 👇

Reply:
That wasn't the goal, it was just to visually structure the code a little bit.